CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Dradis Framework 4.x.x (Month, 2021) ##
+
+* Add CVSS v3.1
```

Comment:
We should mention the toggle exists. When this changelog entry gets bumped over to DradisPro / CE I think it's valuable to mention it now supports both. Long winded rarely hurts in the changelog.

Reply:
updated